src/sign-in-blazorserver/README.md

```diff
@@ -0,0 +1,114 @@
+|--------------------------------:|:--------------------------------------------------------------------|:--------------------------------------------------------------------------------|
+| **Name**                        | `active-directory-dotnet-blazorserver`                              | Suggested value for this sample. <br/> You can change the app name at any time. |
+| **Supported account types**     | **Accounts in this organizational directory only (Single tenant)**  | Suggested value for this sample.                                                |
+| **Platform type**               | **Web**                                                             | Required value for this sample                                                  |
+| **Redirect URIs**               | `http://localhost:5000/signin-oidc`                                 | Required value for this sample                                                  |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Open the project in your code editor
+
+Next, open the _Api.csproj_ project under the the sign-in-blazorserver folder in your code editor.
+
+### 3. Update application code with app registration values
+
+In _appsettings.json_, update each variable with values from the app registration you created earlier.
+
+```json
+"TenantId": "[Enter 'common', or 'organizations' or the Tenant ID (Obtained from the Azure portal. Select 'Endpoints' from the 'App registrations' blade and use the GUID in any of the URLs), e.g. da41245a5-11b3-996c-00a8-4d99re19f292]",
+"ClientId": "[Enter the Client Id (Application ID obtained from the Azure portal), e.g. ba74781c2-53c2-442a-97c2-3d60re42f403]",
+"ClientSecret": "[Copy the client secret added to the app from the Azure portal]",
+```
+
+## Run the application
+
+### 1. Run the webapp
+
+1. Execute the following command to get the app up and running:
+
+   ```bash
+   dotnet run
+   ```
+
+### 2. Signin into the Blazor Server application
+
+1. Once the web app is listening, navigate to https://localhost:5001. Click the Sign In button and complete the login process.
+
+   ![A screenshot of a Blazor Server application application guiding the user to click the "Sign In" button.](./app-signed-in.png)
+
+1. Click Sign Out
+
+   ![A screenshot of a Blazor Server application application guiding the user to click the "Sign Out" button.](./app-signed-out.png)
+
+
+## About the code
+
+This ASP.NET Core 6 Blazor Server application is created using .NET Blazor Server App template. The app is adding sign-in to protect itself, and as a consequence this is requiring the user to be authenticated in Azure Active Directory.
```

Comment:
```suggestion
This ASP.NET Core 6 Blazor Server application is created using the .NET Blazor Server App template. The app is demonstrating the use of Azure Active Directory authentication for user sign-in.
```

Reply:
Please let accept this suggestion partially, and let the rest for later content dev editorial. 

```suggestion
This ASP.NET Core 6 Blazor Server application is created using the .NET Blazor Server App template. The app is adding sign-in to protect itself, and as a consequence this is requiring the user to be authenticated in Azure Active Directory.
```